navsat_transform:
  frequency: 10
  delay: 1.0
  magnetic_declination_radians: 0.01505
  yaw_offset: 2.40000
  zero_altitude: true
  broadcast_utm_transform: false
  publish_filtered_gps: true
  use_odometry_yaw: false
  wait_for_datum: false
